packages/lesswrong/components/ea-forum/giving-portal/EAGivingPortalPage.tsx

```diff
@@ -315,22 +318,8 @@ const EAGivingPortalPage = ({classes}: {classes: ClassesType}) => {
     flash(`Notifications ${notifyForVotingOn ? "disabled" : "enabled"}`);
   }, [currentUser, openDialog, setNotifyForVotingOn, notifyForVotingOn, flash, updateCurrentUser]);
 
-  const onAddCandidate = useCallback(() => {
-    // TODO: Hook up notifications
-    // eslint-disable-next-line no-console
-    console.log("Clicked add candidate");
-  }, []);
-
   const onContribute = useCallback(() => {
-    // TODO: Hook up contribute button
-    // eslint-disable-next-line no-console
-    console.log("Clicked contribute to the discussion");
-  }, []);
-
-  const onLoadMoreOpportunities = useCallback(() => {
-    // TODO: Hook up loading more opportunities
-    // eslint-disable-next-line no-console
-    console.log("Clicked load more donation opportunities");
+    history.push({pathname: '/newPost', search: `?subforumTagId=${donationElectionTagId}`})
```

Comment:
I'm not sure if we want to set the `donationElectionTagId` or the `effectiveGivingTagId` here. We prob only want to do the former if we are keeping that tag and will make it a core tag during the event.

Reply:
I think we want:
 - The link in the "Discuss" card to go to the donation election tag
 - The link in the "Posts tagged "Effective giving"" section to go to the effective giving tags
 - Both of these links to just go to the topic page rather than opening `/newPost` directly (because directly writing a post from this page is a pretty big action, and there are lower effort ways of engaging like commenting)

The `/newPost` thing I think we have discussed in a meeting at some point. The two different links thing is new but I think it makes sense given where the buttons are